### DISCUSSION TITLE
Is there something I just don't get about the ATragMX?

### DISCUSSION THREAD

**Discussion by gronank** [acemod/ACE3#9626] [Q&A]
I've been lóoking into how to use the ATragMX handheld ballistics computer, and after some tinkering, I think the numbers look about right.

There is one thing that befuddles me however: wind direction. It's apparently inputted as a clock direction. The problem with this is the lack of precision. If the wind is blowing 8ms and is 16 degrees off your nose, the crosswind component is 2.2ms. If it's 44 degrees off your nose, the crosswind component is 5.5. That is a 3mil difference at 1km range even though they both register as "1 o'clock". Meanwhile, there is coriolis effects calculated with orders of magnitude smaller impact.

Given that you already input magnetic bearing to the target, it would seem simple enough to me to simply input wind direction as a magnetic bearing as well, particularly since the kestrel straight up gives you that number.

This mod has been around for close to a decade and this is still how it works, so I'm sure I must be missing something.

Kind regards
Gronank

**Comment by Laid3acK** [comment]
The ACE3 AtragMx is configured as the real one for wind direction:
> Wind Direction is expressed in clock points. This illustration shows how to determine the direction. Wind direction always uses the ‘axis of the barrel bore’ as its frame of reference, hence the wind direction is a ‘relative’ direction rather than an absolute direction. Within this ‘axis of the bore’ reference system, wind is always described in terms of where it is coming from.
[Horus manual p.16]

https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf

**Comment by Laid3acK** [comment]
Another important point about wind, Advanced Ballistics has an "advanced wind" which looks like that:
https://wind-data.ch/tools/profile.php?lng=en
The Kestrel will give you wind effects at your position only. Except short range, eventually middle range, global wind effects during bullet's flight will be probably (surely) different. For long range shooting, an estimation of the average wind effects will give better result. Players are also the possibility to enter a min and max wind values to the AtragMx (Horus manual p.32).
